lib/index.d.ts

```diff
@@ -0,0 +1,43 @@
+import {Request} from 'express';
```

Comment:
This module requires types from Express and Passport so those are imported. This import leans on the dependencies of the project using these types so we don't need to import the modules into this project, this is just type definitions, not actual code remember.